cmd/machine-config-operator/bootstrap.go

```diff
@@ -62,6 +63,7 @@ func init() {
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.configFile, "config-file", "", "ClusterConfig ConfigMap file.")
 	bootstrapCmd.MarkFlagRequired("config-file")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.infraConfigFile, "infra-config-file", "/assets/manifests/cluster-infrastructure-02-config.yml", "File containing infrastructure.config.openshift.io manifest.")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.nodeConfigFile, "node-config-file", "/assets/manifests/cluster-node-02-config.yml", "File containing node.config.openshift.io manifest.")
```

Comment:
I'm not sure if you are trying to make this mandatory or not. If you are, you would need to add this in the installer generated manifests first. I'd cross-check with the installer team to see if they are ok with this path.

Since this is for testing purposes, you can just `openshift-install create manifests` -> add that file -> `openshift-install create cluster` to have it render your manually generated one.

Reply:
Yes, I'm trying to figure the places where I need to make the changes.
https://github.com/openshift/installer/pull/5676 is a related draft PR that I created for the testing purpose.
If everything works fine, I would make the proper changes along with the required test cases.
Could you help here in listing out what all other changes needed in introducing this file?